### ISSUE TITLE
Batch update using POST doesn't work anymore.

### ISSUE THREAD

**Issue by jasper-ter-veen** [tailflow/laravel-orion#205]
In the BaseController::retrieve() method change
```php
        if (!config('orion.use_validated')) {
            return $key ? $request->get($key, $default) : $request->all();
        }
```

to 
```php
        if (!config('orion.use_validated')) {
            return $key ? $request->input($key, $default) : $request->all();
        }
```

to fix this

**Comment by alexzarbn** [comment]
The fix is rolled out as part of the [v2.19.1](https://github.com/tailflow/laravel-orion/releases/tag/2.19.1) release.

Thank you for raising the issue!

**Comment by jasper-ter-veen** [comment]
No problem. Thank you for applying the fix quickly!
